packages/smooth_app/lib/pages/product/summary_card.dart

```diff
@@ -390,4 +376,57 @@ class _SummaryCardState extends State<SummaryCard> {
     }
     return result;
   }
+
+  Widget _buildProductQuestionsWidget() {
+    final AppLocalizations appLocalizations = AppLocalizations.of(context)!;
+    if (widget.productQuestions == null) {
+      return EMPTY_WIDGET;
+    }
+    return FutureBuilder<List<RobotoffQuestion>>(
+        future: widget.productQuestions,
+        builder: (
+          BuildContext context,
+          AsyncSnapshot<List<RobotoffQuestion>> snapshot,
+        ) {
+          final List<RobotoffQuestion> questions =
+              snapshot.data ?? <RobotoffQuestion>[];
```

Comment:
What about a `CircularProgressIndicator` while waiting?
What if an error occurs?

Reply:
since this is a tiny widget embedded inside the summary card, I don't think the circular progress indicator makes sense (the user won't know what it's for), if there's an error we just won't show the question prompt. The user still sees the product as they would before this change